webhooks-extension/test/presubmit-tests.sh

```diff
@@ -26,6 +26,9 @@
 export DISABLE_MD_LINTING=1
 export TEST_FOLDER=$(pwd)
 
+# TODO: move to Go modules.
```

Comment:
could you update this TODO with a bit more info about why this fails things and/or maybe link to an issue?

Reply:
Rephrased. It's not so much an issue moreso this package uses Godep, and as of Go 1.16 modules are on by default. This is really here for compatibility.

Reply:
i find that TODO type comments end up just kinda hanging around forever and often eventually lose meaning so i feel like if this is a TODO it would make sense to link to a github issue or something for context (e.g. like `TODO(#11111)`) or remove the "TODO" entirely and just leave something like what you just said, i.e. "this package uses Godep, and as of Go 1.16 modules are on by default"

> This is really here for compatibility.

I don't totally understand why you mean by that, compatibility with what? (maybe the comment could just provide more context)

Reply:
🤦  My git push was stuck on ssh-agent password so it never pushed. *Now* it should be changed.

Reply:
ahhh haha sorry i had a feeling, should have messaged you